src/components/common/TmLiSession.vue

```diff
@@ -20,14 +23,18 @@ export default {
   props: {
     icon: {
       type: String,
-      required: true
+      default: ``
+    },
+    img: {
+      type: String,
+      default: ``
     },
     title: {
       type: String,
       required: true
     },
     route: {
-      type: String,
+      type: String || Object, // this is not working. Where is type any
```

Comment:
there is a way to have `or` in types - but it smells like a bad design

Reply:
```suggestion
      type: String
```

Reply:
It will still throw an error, since I am sending an Object from `TmSelectNetwork`...

Reply:
but you shouldn't

Reply:
send an Object you mean? I could stringify it, yes, but what happens when we do send a string? :woman_shrugging:

Reply:
no you should not send an object at all. you should set an id and then query for the object in the route / component that you want the object to access from

Reply:
I fixed it a bit differently now. You can have a look. Thanks to @jbibla suggestion, I now kill two birds with one shot (WARNING: directly translated idiom):

When the user selects the network in TmSelectNetwork, they get connected to that network. So when I am creating the address in TmSessionSignUpSeed, I just grab `connection` and that it. Fixes passing along the info and getting connected to the right network in one shot :wink:

Reply:
kill two birds with one ~shot~ stone 😉

Reply:
what is the flow here, if you start the flow and the stop? you are now connected to the target network. isn't that confusing?